sphinx_intl/transifex.py

```diff
@@ -63,11 +57,64 @@ def normalize_resource_name(name):
 
     return name
 
+def check_transifex_cli_installed():
+    if not which("tx"):
+        msg = textwrap.dedent("""\
+            Could not run "tx".
+            You need to install the Transifex CLI external library.
+            Please install the below command and restart your terminal if you want to use this action:
+
+                $ curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+
+            """)
+        raise click.BadParameter(msg)
+
+    version_msg = subprocess.check_output("tx --version", shell=True).decode('utf-8')
+    version = tuple(int(x) for x in version_msg.split("=")[-1].strip().split("."))
+
+    if not version_msg.startswith("TX Client"):
+        msg = textwrap.dedent("""\
+            The old transifex_client library was found.
+            You need to install the Transifex CLI external library.
+            Please install the below command and restart your terminal if you want to use this action:
+
+                $ curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+
+            """)
+        raise click.BadParameter(msg)
+
+    if not version >= MINIMUM_VERSION:
+        msg = textwrap.dedent(f"""\
+        An unsupported version of the Transifex CLI was found.
+        Version {MINIMUM_VERSION} or greater is required.
+        Please run the below command if you want to use this action:
+
+            $ tx update
+
+        """)
+        raise click.BadParameter(msg)      
+
+ 
+def get_tx_root():
+    check_transifex_cli_installed()
```

Comment:
It seems `check_transifex_cli_installed()` is already called inside `update_txconfig_resources()`. So this is not needed here.

Additionally, it would be better to check this explicitly. So it's not good to call this automatically on `get_tx_root()`.

Reply:
I just got rid of `get_tx_root()` as it's not being used anywhere with the go library